libcontainer/factory_nabla.go

```diff
@@ -86,6 +87,17 @@ func applyPauseHack(config *configs.Config, containerRoot string) (*configs.Conf
 func createRootfsISO(config *configs.Config, containerRoot string) (string, error) {
 	rootfsPath := config.Rootfs
 	targetISOPath := filepath.Join(containerRoot, "rootfs.iso")
+	for _, mount := range config.Mounts {
+		if (mount.Destination == "/etc/resolv.conf") ||
+			(mount.Destination == "/etc/hosts") ||
+			(mount.Destination == "/etc/hostname") {
+			cmd := exec.Command("cp", mount.Destination, rootfsPath + "/etc/")
```

Comment:
- Hmm, should this be copying from the `mount.Source` instead of the destination? I believe these files that are populated are specific to the container. 

- For destination, would recommend using [`filepath.Join`](https://golang.org/pkg/path/filepath/#Join)

- Also, we have wrote this nice utility function that you can use [here :) ](https://github.com/nabla-containers/runnc/blob/af7e9863dd2414cee95cfb94b8e7e63949208fc5/utils/copy.go)